### DISCUSSION TITLE
Currency Tab confusion

### DISCUSSION THREAD

**Discussion by SodaSyrup** [dickwolff/Export-To-Ghostfolio#108] [Q&A] [answered]
Hello, i just added GBP as a currency and something odd is happening with it...

![image](https://github.com/user-attachments/assets/7e88dde3-83b9-43bc-974b-d6f9c665aa3f)

**Comment by dickwolff** [comment, marked as answer]
If you mean 2 type of GBP/p, the main difference as I know it is the small p (`GBp`) is in pence, which is a hundreth of a pound (£, `GBP`).

**Reply by SodaSyrup** [reply]
oh ok, thanks didnt know that one lol
